code/modules/clothing/head/misc_special.dm

```diff
@@ -218,3 +218,22 @@
 	hair_style = pick(GLOB.hair_styles_list - "Bald") //Don't want invisible wig
 	hair_color = "#[random_short_color()]"
 	. = ..()
+
+/obj/item/clothing/head/foilhat
+	name = "tinfoil hat"
+	desc = "Thought control rays, psychotronic scanning. Don't mind that, I'm protected cause I made this hat."
+	icon_state = "foilhat"
+	item_state = "foilhat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = -5,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = -5, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user)
+	user.gain_trauma(/datum/brain_trauma/mild/phobia, FALSE, "security", "lizards")

```

Comment:
What if you are security or lizard?

Reply:
then you either don't put on the hat or you do and crumble inward as you realize everything you've ever stood for is wrong and a lie.

Reply:
that's... deep

Reply:
Either that or having a loyalty implant nullifies both the conspiracy revelation and ayy blocking aspects of tinfoil hats.
The conspiracy's already got you, mate, no amount of tinfoil is going to help you now that the government's reptillian alien globalists has a whole bunch of their tech jammed into yer brain!